src/button.test.accessibility.ts

```diff
@@ -2,10 +2,22 @@ import { html } from 'lit';
 import { expect, test } from './playwright/test.js';
 import type Button from './button.js';
 
+test('is accessible', { tag: '@accessibility' }, async ({ mount, page }) => {
+  await mount(
+    () =>
+      html`<glide-core-button
+        label="Label"
+        aria-description="Description"
+      ></glide-core-button>`,
+  );
+
+  await expect(page).toBeAccessible('glide-core-button');
+});
+
 test(
-  'is accessible',
```

Comment:
Broke this test down into separate tests now that there's no longer a benefit to cramming them into `"is accessible"`.